src/main/scala/org/neo4j/spark/Neo4jOptions.scala

```diff
@@ -48,12 +48,7 @@ class Neo4jOptions(private val parameters: java.util.Map[String, String]) extend
       Neo4jQueryOptions(LABELS, parsed)
     }
     case ("", "", relationship) => Neo4jQueryOptions(RELATIONSHIP, relationship)
-    case _ => throw new IllegalArgumentException(
-      s"You need to specify just one of these options: ${
-        QueryType.values.toSeq.map(value => s"'${value.toString.toLowerCase()}'")
-          .sorted.mkString(", ")
-      }"
-    )
+    case _ => throw new IllegalArgumentException("You must specify one of `query`, `labels`, `relationship`")
```

Comment:
why this change?

Reply:
This was unwanted. I'll restore it. It doesn't change much much but I prefer the previous one